src/less/icon/foreground/ds6/foreground.less

```diff
@@ -306,4 +306,16 @@ svg {
     &.icon--window {
         .icon-size-mixin('icon-window');
     }
+
+    &.icon--confirmation {
+        .icon-size-mixin('icon-confirmation-foreground');
+    }
+
+    &.icon--information {
+        .icon-size-mixin('icon-information-foreground');
+    }
+
+    &.icon--attention {
+        .icon-size-mixin('icon-attention-foreground');
```

Comment:
Looks like these will change when you change the variable names elsewhere. Just wanted to note it to make sure it's addressed. 👍

Reply:
Whoops, didn't realize. Let me address this.

Reply:
Fixed/resolved